linux/files/90autoupdates

```diff
@@ -0,0 +1,29 @@
+{%- set autoupdates = salt['pillar.get']('linux:system:autoupdates') %}
+
+{%- if autoupdates.enabled %}
```

Comment:
but the default value for `enabled` is already set in `map.jinja`, isn't it?
btw, I had set the default to `False`, do you want to change it to `True`?